README.md

```diff
@@ -1,6 +1,6 @@
 # Bonsai Gym
 
-A python library for integrating Bonsai BRAIN with Open AI Gym environments.
```

Comment:
think we are standardizing away from the [ALL CAPS](https://www.youtube.com/watch?v=ewc1hixzYPY&ab_channel=StonesThrow) terminology.